### DISCUSSION TITLE
Add cmi.command.hologram.own permission

### DISCUSSION THREAD

**Discussion by mzyax** [Zrips/CMI#3388] [Feature Requests]
**Description of feature/improvement request:**
Extends the use of holograms from admin only to allowing players to use holograms.

---
### Detailed explanation of what it should do, when and in what situation: ###
Giving cmi.command.hologram.own allows the player to create and edit and remove their own hologram. 

### Basic explanation as to why this feature is needed and/or useful to have: ###
It's admin only and could be more useful for the average player. For example, this would allow a player to create a hologram in their towny town for better instructions for their residents.

**Comment by Zrips** [comment]
Issue comes with holograms itself, they are somewhat heavy on client (not server) so we would need to add not only basic permission node, but we would need to track who owns particular hologram, implement more harsh restrictions on how far you can see it, how fast it updates, hoe many lines it could have an so on.
